docs/helpers/env_vars.yaml

```diff
@@ -1,3 +1,104 @@
+ACTIVITYLOG_SERVICE_ACCOUNT_ID:
```

Comment:
Why are there various new env vars with `introductionVersion: "5.0"` ?
5.0 was released a while ago, so I expected that these would already exist in the docs.